### PR TITLE
Add Github Omniauth

### PR DESCRIPTION
**Pull request by tiagocasanovapt**
## Context

This depends on #2, notice the base branch is not `main`

## Description

- Installs `omniauth` & `omniauth-github`
- Copied the HTML Devise templates from `points` app
- Copied the `CallbacksController` from `points` app

## Notes

There might be issues using it in production as noted in the [Devise wiki](https://github.com/heartcombo/devise/wiki/How-To:-Use-devise-inside-a-mountable-engine#omniauth), but there are workarounds.

## Issue

Related with https://github.com/fastruby/ombu_labs-auth/issues/1#issuecomment-1277877820

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,3 +12,5 @@ gem "sprockets-rails"
 # gem "debug", ">= 1.0.0"
 
 gem "devise"
+gem "omniauth"
+gem "omniauth-github"
```
Comment (etagwerker) [inline Gemfile:16]: @tiagocasanovapt No need to add this here. `gemspec` already defines devise and omniauth and omniauth-github as dependencies and you are including it in line 5.

**Review by etagwerker — CHANGES_REQUESTED** [review]
@tiagocasanovapt just one comment here.

File: test/dummy/db/schema.rb [hunk 2]
```diff
@@ -14,18 +14,6 @@
...
-    t.string "encrypted_password", default: "", null: false
-    t.string "reset_password_token"
-    t.datetime "reset_password_sent_at"
-    t.datetime "remember_created_at"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["email"], name: "index_auth2_users_on_email", unique: true
-    t.index ["reset_password_token"], name: "index_auth2_users_on_reset_password_token", unique: true
-  end
-
```
Comment (tiagocasanovapt) [inline test/dummy/db/schema.rb:28]: leftovers from previous testing

**Review by etagwerker — APPROVED** [review]
@tiagocasanovapt Looks good, thanks!

**Pull request merged**